packages/frontend/src/features/sqlRunner/components/Header/HeaderEdit.tsx

```diff
@@ -0,0 +1,184 @@
+    const savedSqlChart = useAppSelector(
+        (state) => state.sqlRunner.savedSqlChart,
+    );
+    const sql = useAppSelector((state) => state.sqlRunner.sql);
+    const config = useAppSelector((state) =>
+        state.sqlRunner.selectedChartType === ChartKind.TABLE
+            ? state.tableVisConfig.config
+            : state.barChartConfig.config,
+    );
+    const { mutate } = useUpdateSqlChartMutation(
+        savedSqlChart?.project.projectUuid || '',
+        savedSqlChart?.savedSqlUuid || '',
+    );
+
+    const isSaveModalOpen = useAppSelector(
+        (state) => state.sqlRunner.modals.saveChartModal.isOpen,
+    );
+    const onCloseSaveModal = useCallback(() => {
+        dispatch(toggleModal('saveChartModal'));
+    }, [dispatch]);
+    const isDeleteModalOpen = useAppSelector(
+        (state) => state.sqlRunner.modals.deleteChartModal.isOpen,
+    );
+    const onCloseDeleteModal = useCallback(() => {
+        dispatch(toggleModal('deleteChartModal'));
+    }, [dispatch]);
+    const isUpdateModalOpen = useAppSelector(
+        (state) => state.sqlRunner.modals.updateChartModal.isOpen,
+    );
+    const onCloseUpdateModal = useCallback(() => {
+        dispatch(toggleModal('updateChartModal'));
+    }, [dispatch]);
+
+    if (!savedSqlChart) {
+        return null;
+    }
+
+    return (
+        <>
+            <Paper shadow="none" radius={0} px="md" py="xs" withBorder>
+                <Group position="apart">
+                    <Stack spacing="none">
+                        <Group spacing="two">
+                            <TitleBreadCrumbs
+                                projectUuid={savedSqlChart.project.projectUuid}
+                                spaceUuid={savedSqlChart.space.uuid}
+                                spaceName={savedSqlChart.space.name}
+                            />
+                            <Title c="dark.6" order={5} fw={600}>
+                                {savedSqlChart.name}
+                            </Title>
+                        </Group>
+                        <Group spacing="xs">
+                            <UpdatedInfo
+                                updatedAt={savedSqlChart.lastUpdatedAt}
+                                user={savedSqlChart.lastUpdatedBy}
+                                partiallyBold={false}
+                            />
+                            <ResourceInfoPopup
+                                resourceUuid={savedSqlChart.savedSqlUuid}
+                                projectUuid={savedSqlChart.project.projectUuid}
+                                description={
+                                    savedSqlChart.description ?? undefined
+                                }
+                                viewStats={1} // todo: update endpoint to return view stats
+                                firstViewedAt={undefined}
+                                withChartData={false}
+                            />
+                        </Group>
```

Comment:
The rest of the code was copied from `Header` component. This is the new part that has the chart details.